Integrations/integration-RsaNetwitnessSecurityAnalytics.yml

```diff
@@ -1081,57 +1297,60 @@ script:
         case 'nw-login':
             results = sessionId;
             break;
-        case 'nw-list-incidents':
+        case 'netwitness-im-list-incidents':
             var incidents = listIncidents(sessionId, args, incidentManagementId);
-            results = { incidents: incidents };
+            results = buildRetValListIncidents(incidents);
             break;
-        case 'nw-get-incident-details':
+        case 'netwitness-im-get-incident-details':
             var incident = getIncidentById(sessionId, args, incidentManagementId);
-            results = { incident: incident };
+            results = buildRetValGetIncidentDetails(incident);
             break;
-        case 'nw-get-components':
+        case 'netwitness-im-get-components':
             var components = getComponents(sessionId, args);
-            results = { components: components };
+            results = buildRetValGetComponents(components);
             break;
-        case 'nw-get-events':
+        case 'netwitness-im-get-events':
             var events = getEvents(sessionId, args);
-            results = { events: events };
+            results = buildRetValGetEvents(events);
```

Comment:
When I execute `!netwitness-im-get-events timeRangeType="ALL_DATA" deviceId="6"`
`==` shouldn't be there

![image](https://user-images.githubusercontent.com/7270217/28249258-d1f691ac-6a5a-11e7-952b-b63cc53897b8.png)